code/config/otel.yml

```diff
@@ -18,12 +24,23 @@ exporters:
     endpoint: ${env:LOKI_URL}
     tls:
       insecure: true
+  prometheus:
+    endpoint: 0.0.0.0:8889
+
+processors:
+  batch:
 
 service:
   pipelines:
     logs:
       receivers: [otlp]
-      exporters: [loki, otlp/dashboard, debug]
+      processors: [batch]
+      exporters: [loki,debug]
+    metrics:
+      receivers: [prometheus,otlp]
+      processors: [batch]
+      exporters: [prometheus,debug]
     traces:
       receivers: [otlp]
-      exporters: [otlp, otlp/dashboard, debug]
```

Comment:
This needs to be added back if we also want the Aspire dashboard to represent the otel information.